helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/StoppableInstancesSelector.java

```diff
@@ -118,10 +152,22 @@ public void getStoppableInstancesCrossZones() {
    *
    * @param random Indicates whether to randomize the order of zones.
    */
-  public void calculateOrderOfZone(boolean random) {
+  public void calculateOrderOfZone(List<String> instances, boolean random) {
```

Comment:
I don't see any place to use this random

Reply:
At line 177, it's used to determine whether we want to do the shuffling of _orderOfZone. if random:`Collections.shuffle(_orderOfZone);`